script/results.html

```diff
@@ -63,6 +153,426 @@
+  buttonElementId = "feedback-tab"
+  enableSendDownload()
+  document.getElementById("optional-tab").setAttribute("hidden", "hidden");
+  document.getElementById("optional-tab-telco").setAttribute("hidden", "hidden");
+  document.getElementById("optional-tab-non-telco").setAttribute("hidden", "hidden");
+  document.getElementById("myCheck1").setAttribute("hidden", "hidden");
+  document.getElementById("myCheck2").setAttribute("hidden", "hidden");
+  document.getElementById("myCheck3").setAttribute("hidden", "hidden");
+  var selectBox = document.getElementById("selectBox");
+    var selectedValue = selectBox.options[selectBox.selectedIndex].value;
+    if (selectedValue=="all"){
+      showAll()
+    }
+    if (selectedValue=="faredge"){
+          document.getElementById("myCheck1-feedback").removeAttribute("hidden");
+          document.getElementById("optional-tab-feedback").removeAttribute("hidden");
+          document.getElementById("myCheck2-feedback").setAttribute("hidden", "hidden");
+          document.getElementById("myCheck3-feedback").setAttribute("hidden", "hidden");
+          document.getElementById("optional-tab-non-telco-feedback").setAttribute("hidden", "hidden");
+          document.getElementById("optional-tab-telco-feedback").setAttribute("hidden", "hidden");
+          showFarEdge()
+
+    }
+    if (selectedValue=="telco"){
+      document.getElementById("myCheck1-feedback").setAttribute("hidden", "hidden");
+          document.getElementById("optional-tab-feedback").setAttribute("hidden", "hidden");
+          document.getElementById("optional-tab-non-telco-feedback").setAttribute("hidden", "hidden");
+          document.getElementById("myCheck3-feedback").setAttribute("hidden", "hidden");
+          document.getElementById("optional-tab-non-telco-feedback").setAttribute("hidden", "hidden");
+
+          document.getElementById("myCheck2-feedback").setAttribute("hidden", "hidden");
+          document.getElementById("optional-faredge-feedback").setAttribute("hidden", "hidden");
+
+        showTelco()
+
+
+          document.getElementById("myCheck3-feedback").removeAttribute("hidden");
+          document.getElementById("optional-tab-telco-feedback").removeAttribute("hidden");
+
+    }
+    if (selectedValue=="nontelco"){
+          document.getElementById("myCheck3-feedback").setAttribute("hidden", "hidden");
+          document.getElementById("myCheck1-feedback").setAttribute("hidden", "hidden");
+          document.getElementById("optional-tab-feedback").setAttribute("hidden", "hidden");
+          document.getElementById("optional-tab-telco-feedback").setAttribute("hidden", "hidden");
+
+          document.getElementById("myCheck2-feedback").removeAttribute("hidden");
+          document.getElementById("optional-tab-non-telco-feedback").removeAttribute("hidden");
+          showNonTelco()
+    }
+}
+
+function changeFunc() {
+    var selectBox = document.getElementById("selectBox");
```

Comment:
The test summary always show the summary for the total number of test. this should update based on the scenario selected

Reply:
The test count seems to be wrong when optional checkbox is unchecked